### ISSUE TITLE
Doesn't the newer versions of todo.txt have priority options.

### ISSUE THREAD

**Issue by gooddadmike** [jdiamond/todo.txt-ahk#2]
I think with the new todo.txt touch version a compiled version of this that is up to date could get very popular if it were added to todotxt.com and it would need priority option and color coding. As a matter of fact why not use the same color scheme for continuity?

**Comment by jdiamond** [comment]
I've never used the "touch" version so am not aware of the coloring scheme that it uses. I can look into seeing if AHK makes it easy to color rows in list views. If it is, I'd be happy to update it, but probably not until this weekend.

**Comment by gooddadmike** [comment]
You should look at the latest Cygwin Version for the coloring to make sure
it is the same and also maybe do a final comparison to the output files to
make sure they will be cross compatible because you know people will try
this app with existing todo.txt files they have already been using.

You rock man! Thanks!
Alion Science and Technology
502-252-3588
mikesearcy.com

**Comment by jdiamond** [comment]
I'll take a look at how they specify the priority option. It's not a feature I use so didn't look for it. If coloring is easy, it might be easy to let the colors be configured in todo.ini, but default to the same colors the CLI uses.

**Comment by jdiamond** [comment]
Looked for how AHK allows you to set colors on rows in a list view and it doesn't support it, but there are some scripts in the forums that allow you to do this so it should be possible, but I haven't been able to find the time to learn how to use them yet.

**Comment by gooddadmike** [comment]
Have you seen the Todo.txt Touch application. It has a really nice layout.
 You should look at emulating it and it also included the priority stuff I
was talking about before.
Alion Science and Technology
502-252-3588
mikesearcy.com

**Comment by jdiamond** [comment]
I've seen the screenshots, but don't have an Android to actually run it. It looks like it might work well with a touch screen, but I'm not sure how well it would translate to desktop usage with a full keyboard and being able to tab from control to control. I can definitely make it detect priorities and add a new column to make it easy to sort by them, but I prioritize in my head so it's not a feature I'm dying to add.

**Comment by gooddadmike** [comment]
Right but full complaince with other versions will be essential for
widespread use.  They won't want to use your version on the desktop if their
android version makes a todo.txt files that comes up garbled in the AHK app.
Alion Science and Technology
502-252-3588
mikesearcy.com

**Comment by jdiamond** [comment]
I agree that the data in the todo.txt file needs to follow the format defined on the todotxt.com website, but that's different from how it gets displayed. As far as I know, I don't produce anything that can't be read by the CLI or Android apps. Actually, I allow spaces in contexts and projects and I'm not sure the CLI can handle that.

Priority seems to be just the presence of "(X)" where X is A, B, or any other letter on the line in the todo.txt file. That's why I think it'd be easy to add explicit support for it. Even without explicit support, you can always stick "(A)" at the beginning of an item and then sort by description and that could work.

**Comment by notbalanced** [comment]
Hi, I really like your AHK implementation of the todo.txt concept.  I have forked the project and made some changes, including adding a priority field and color to the list items.  You're welcome to take a look at the changes. The color stuff is in a separate file LVCustomColors.ahk, so might be easy to pull in.
